src/scss/components/optional/_modal.scss

```diff
@@ -139,6 +139,7 @@ $modal-borderRadius     : 4px;
             }
 
         .c-modal-content-scrollable {
+            max-height: 100vh;
```

Comment:
Have we checked how this looks visually on iOS devices as `100vh` extends below the bottom bar? It might be worth looking at `max-height: -webkit-fill-available;` with the appropriate `100vh` fallback.

Reference: [CSS3 100vh not constant in mobile browser](https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser).

Reply:
Will test that out in another PR, this one is to revert a change which has broken a consumer of this style and is blocking a release.

Reply:
We've hit all sorts of issues with mobile heights on different devices and OSs due in most part to the browser UI which they add in, such as the nav bar at the bottom in iOS, that doesn't respect the 100vh values correctly.

Reply:
Yeah it's a royal pain. Sounds good though, thanks for clarifying :)